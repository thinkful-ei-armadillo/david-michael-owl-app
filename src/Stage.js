import React from 'react'
import Store from './STORE'


function Stage (){

  const stagePeople = Store.participants.filter((person)=> person.onStage === true);

  return (
    <div className="Stage">


    {stagePeople.map((person) => 
      
        <div key ={person.id}> 
          <img src = {person.avatar} alt="chat icon"/>
            <p>{person.name}</p>
      </div>
)}
    </div>
  )
}


export default Stage;