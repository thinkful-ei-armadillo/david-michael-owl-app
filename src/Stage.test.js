import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import Stage from './Stage'

describe('<Stage />', () => {
    it('renders without crashing', ()=> {
        const div = document.createElement('div');
        ReactDOM.render(<Stage avatar ='Our header' name = 'Bob' />, div);
        ReactDOM.unmountComponentAtNode(div);
      })
    })



describe('<Stage />', () => {
  it('renders without crashing', () => {
      // Render the component, as JSON
      const tree = renderer.create(<Stage avatar ='Our header' name = 'Bob' />).toJSON();
      // Check whether it matches the previous snapshot
      // Stored in __snapshots__/App.test.js.snap
      expect(tree).toMatchSnapshot(); 
  });
});
