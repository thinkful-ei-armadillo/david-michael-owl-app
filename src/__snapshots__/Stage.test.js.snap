// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stage /> renders without crashing 1`] = `
<div
  className="Stage"
>
  <div>
    <img
      alt="chat icon"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <p>
      Koren Templeton
    </p>
  </div>
  <div>
    <img
      alt="chat icon"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
    <p>
      Ashla Attwool
    </p>
  </div>
</div>
`;
